### DISCUSSION TITLE
Add "Autoload" animation tracks to the `AnimationPlayer`

### DISCUSSION THREAD

**Discussion by pixel-boy** [godotengine/godot-proposals#11435] [Engine Core]
I don't think it's possible to edit autoloads directly via the `AnimationPlayer` currently ?

The idea would be to add a track type that allows to edit autoloads via the `AnimationPlayer `.
![track_types](https://github.com/user-attachments/assets/5925012f-b815-4f2c-b43d-8ba77e3188d9)

The current workaround adds an extra step for each property that you want to modify in an Autoload.
In the same subject being able to edit audio buses and bus effects via an `AnimationPlayer` could also be practical.

I don't know if it would really be a good practice to be able to do this, but it doesn't cost much to suggest it.

**Comment by Calinou** [comment]
I don't think this needs a dedicated track type. You just need to be able to target the autoload with any given track type.

This may be feasible already by manually entering the autoload node name after creating an animation track for a different node, e.g. `/root/Something:example` for the property `example` in an autoload named `Something`. If this is proven to work, then we could modify the node picker in the animation editor to have an option to show autoloads.

**Reply by pixel-boy** [reply]
Thanks, it works with that, the only problem is that the `AnimationPlayer ` cannot deduce the type of the variables so it's not possible to add keys directly, I was able to make it work by first putting a property of a node, adding the keys then renaming the property by that of the autoload.
